### PR TITLE
Fix padding for upcoming events and for small screens

### PR DESCRIPTION
**Pull request by yanbhoe**

### CODE REVIEW

**Review by jesseez — APPROVED** [review]
lgtm

File: web/src/components/home/subcomponents/upcoming_events/upcoming_events.js [hunk 1]
```diff
@@ -74,11 +74,12 @@ function UpcomingEvents() {
 
   return (
     <Box sx={upcomingEventsStyles.upcomingEventsContainer}>
-      <Container sx={upcomingEventsStyles.upcomingEventsRendererContainer}>
+      <Container>
         <Typography variant="h4">{messages.upcomingEventsTitle}</Typography>
         {FeatureFlags.showScheduleTab && (
           <Typography>{messages.upcomingEventsDescription}</Typography>
         )}
+        <Typography>Teset</Typography>
```
Comment (Kuboosh21) [inline web/src/components/home/subcomponents/upcoming_events/upcoming_events.js:82]: What does this mean, this isn't a word

Comment (jesseez) [reply web/src/components/home/subcomponents/upcoming_events/upcoming_events.js:82]: ur not a word

Comment (jesseez) [reply web/src/components/home/subcomponents/upcoming_events/upcoming_events.js]: @Kuboosh21 fixed it. Plz lgtm

**Review by doyourownswing — APPROVED** [review]
Wow this is so great guys! Nicely done!

**Pull request merged**